apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -122,7 +154,56 @@ export default function RootStateDrawer() {
     ],
   )
 
-  const pageLayout: string = savedPageState.layout
+  const handleConversionToIndexPage = useCallback(() => {
```

Comment:
sorry, this got swapped downstream because now we are changing to using `blob.content` for the childpages...
 
because of this, you'd need to find the `childpage` block (`type === 'childpage'`) or just use the `currActiveIdx`

Reply:
can rebase onto `ref/index-pages-in-content` for this change :sadge:

Reply:
Rebased!